client/require.js

```diff
@@ -0,0 +1,17 @@
+const moduleStore = {
```

Comment:
Yeah it's same. But there's no eval part. Just return the module.
Shall I make changes to evalScript to support this?
If so, shall we make it as `require.js` rather eval-script :)